test/unit/mocks/entities.js

```diff
@@ -509,6 +509,22 @@ export const scheduledActionCollectionMock = {
   items: [cloneDeep(scheduledActionMock)],
 }
 
+const entryWithReferencesMock = {
+  sys: {
+    type: 'Array',
+  },
+  items: [makeLink('Entry', 'entry-id')],
```

Comment:
might be misleading because the items are not just links btw (they are the entry fields, sys, metadata etc) 🤔 

maybe you could use the entryMock that is in this file?

Reply:
updated